client/test/e2e/shared/components.js

```diff
@@ -134,6 +134,15 @@ exports.currencyInput = {
   },
 
   /**
+  *set the value of the component by his id to avoid conflict
+  */
+  setById : function setById (id, value){
```

Comment:
This solution is not efficient when looking element by css, it will send back two or more elements  and protractor will choose the first one, it is unsafe.
The alternative should be to identify a component by ```data-*``` or ```id``` in the page which use the component but not to touch the component directly.